src/api/services/NotificationAPI.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * NotificationAPI Services
```

Comment:
consider that this code will be auto-generated from the pagopa swagger specs
cc @cloudify